MAINTAINERS.md

```diff
@@ -0,0 +1,51 @@
+# Helm Maintainers
+
+This document explains the leadership structure of the Kubernetes Helm project, and list the current project maintainers.
+
+The Helm project welcomes contributions from any interested parties. This document explains the role of what we call "Core Maintainers". Core Maintainers play the special role of curating the project.
+
+## What is a Core Maintainer?
+
+(Unabashedly stolen from the [Docker](https://github.com/docker/docker/blob/master/MAINTAINERS) project)
+
+There are different types of maintainers, with different responsibilities, but
+all maintainers have 3 things in common:
+
+1. They share responsibility in the project's success.
+2. They have made a long-term, recurring time investment to improve the project.
+3. They spend that time doing whatever needs to be done, not necessarily what
+is the most interesting or fun.
+
+## Helm Core Maintainers
+
+The duties of a core maintainer include:
+
+* Classify and respond to GitHub issues and review pull requests
+* Perform code reviews (see below)
+* Shape the Helm roadmap and lead efforts to accomplish roadmap milestones
+* Participate actively in feature development and bug fixing
+* Answer questions and help users
+
+The current core maintainers of Helm (in alphabetical order):
+
+* Ville Aikas - [@vaikas-google](https://github.com/vaikas-google)
+* Matt Butcher - [@technosophos](https://github.com/technosophos)
+* Jack Greenfield - [@jackgr](https://github.com/jackgr)
+* Brendan Melville - [@bmelville](https://github.com/bmelville)
+* Michelle Noorali - [@michelleN](https://github.com/michelleN)
+* Adam Reese - [@adamreese](https://github.com/adamreese)
+
+### Code Reviews
+
+Before being merged, every pull request must have at least two LGTMs ("Looks Good To Me"), and at least one of these must come from one of the core maintainers listed above.
+
+## Becoming a Core Maintainer
+
+Generally, potential core maintainers are selected by the Helm core maintainers based in
+part on the following criteria:
+
+* Sustained contributions to the project over a period of time (usually months)
+* A willingness to help users on GitHub and in the Helm Slack room for [Kubernetes](http://slack.kubernetes.io/)
+* A friendly attitude
+
+The Helm core maintainers must unanimously agree before inviting a community member to join as a maintainer, although in many cases the candidate has already been acting in the capacity of a contributing maintainer for some time, and has been consulted on issues, pull requests, etc.
```

Comment:
I'd be fine with continuing in this manner. 
However, I will respectfully disagree with the idea that if this were a large and bureaucratic organization that we would need a unanimous vote. I think those are more realistic with small teams like ours. I just like the idea of consensus but I do think there is enough trust here to be fine with letting admins add and remove core team members as they and the team see fit.